app/Lib/Export/StixExport.php

```diff
@@ -131,7 +131,11 @@ private function __delete_temporary_files($index)
     {
         foreach ($this->__filenames as $f => $filename) {
             if ($index >= $f) {
-                unlink($this->__tmp_dir . $filename);
+                try {
+                    unlink($this->__tmp_dir . $filename);
+                } catch (Exception $e) {
```

Comment:
@RichieB2B I think `unlink` never throw exception, so this catch will be never called. If you want to suppress error for non exists file, you should use [`@` operator](https://www.php.net/manual/en/language.operators.errorcontrol.php) or check file existence first.

Reply:
Good catch! It is indeed a warning which can be suppressed using the @ operator.
```
$ cat unlink.php 
<?php
unlink('/tmp/foobar');
try {
  unlink('/tmp/barfoo');
} catch (Exception $e) {
}
@unlink('/tmp/foofoo');
$ php unlink.php 
PHP Warning:  unlink(/tmp/foobar): No such file or directory in /tmp/unlink.php on line 2
PHP Warning:  unlink(/tmp/barfoo): No such file or directory in /tmp/unlink.php on line 4
```